src/plugins/action_generate_review_assignment_records/src/generateReviewAssignments.ts

```diff
@@ -45,6 +45,13 @@ module.exports['generateReviewAssignments'] = async function (input: any, DBConn
         ? restrictions?.templateSectionRestrictions
         : null
 
+      const canSelfAssign = restrictions?.canSelfAssign
+      // Automatic option for slef assignment if review is above level 1
+      const status =
+        canSelfAssign || nextReviewLevel > 1
+          ? AssignmentStatus.SELF_ASSIGN
+          : AssignmentStatus.AVAILABLE
+
```

Comment:
This is going to conflict with what @nmadruga's done in #280, so you'll need to resolve that.

Reply:
https://github.com/openmsupply/application-manager-server/pull/283/commits/acb17b21deb1bdee1565bb875ce888e8f8e49605

will need to restest